platforms/documentation/docs/src/docs/userguide/best-practices/bp_general.adoc

```diff
@@ -15,4 +15,47 @@
 [[best_practices_general]]
 = General Gradle Best Practices
 
-Add content here.
+[[use_the_plugins_block]]
+== Apply Plugins Using the `plugins` Block
+
+You should always use the `plugins` block to <<plugin_basics.adoc#applying_plugins,apply plugins>> in your build scripts.
+
+=== Explanation
+
+The `plugins` block is the preferred way to apply plugins in Gradle.
+The plugins API allows Gradle to better manage the loading of plugins and it is both more concise and less error-prone than adding dependencies to the buildscript's classpath explicitly in order to use the `apply` method.
+
+It allows Gradle to optimize the loading and reuse of plugin classes and helps inform tools about the potential properties and values in extensions the plugins will add to the build script.
+It is constrained to be idempotent (produce the same result every time) and side effect-free (safe for Gradle to execute at any time).
+
+It is also likely that the `plugins` block will be the only way to apply plugins in the future.
```

Comment:
This seems kind of speculative. If we do plan to do this for real, we should link to a tracking issue, if we don't I think we should remove it.

Reply:
> It is also likely that the `plugins` block will be the only way to apply plugins in the future.

You mean this part, right?  I think it's very safe to say we're all in agreement that we want to do this and will do it _someday_.  But I take your point about avoiding speculation in these items.  As I can't find an issue for it, I'll remove it.

Reply:
Yes that part. Removing sounds good.